qemu/tests/x86_cpu_L3_cache.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_misc
+from virttest import utils_qemu
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check L3 cache present to guest
+
+    1. boot guest with latest machine_type('pc' or 'q35'),
+     checking L3 cache presents inside guest.
+    2. Boot guest with old machine type(rhel7.3.0, pc or q35),
+     L3 cache shouldn't present inside guest.
```

Comment:
please delete the words` pc or q35` since we have separated the step by the kar parameter `--machines`